src/Core/Managed/Core.Net46/Extensibility/Implementation/External/OperationContextData.cs

```diff
@@ -57,38 +57,17 @@ public string Name
             set { this.tags.SetStringValueOrRemove(ContextTagKeys.Keys.OperationName, value); }
         }
 
-        public string ParentId
-        {
-            get { return this.tags.GetTagValueOrNull(ContextTagKeys.Keys.OperationParentId); }
-            set { this.tags.SetStringValueOrRemove(ContextTagKeys.Keys.OperationParentId, value); }
-        }
-
-        public string RootId
-        {
-            get { return this.tags.GetTagValueOrNull(ContextTagKeys.Keys.OperationRootId); }
-            set { this.tags.SetStringValueOrRemove(ContextTagKeys.Keys.OperationRootId, value); }
-        }
-
```

Comment:
I see. So the public contract did not change? We can change it, but if we do, we need to go to 2.0.